### ISSUE TITLE
customer cannot permanently update his postal address when placing an order

### ISSUE THREAD

**Issue by NickWeir63** [openfoodfoundation/openfoodnetwork#1030]
We had this feedback from a Stroudco shopper who has recently moved house:
"every week I correct the address given for me on the Check Out page.  Every week the old address re-appears".  
I tested changing the home address on an order on my own account and it saved the change correctly for the next time i put an order in.  
So I then reset the password on the shopper's account and tried changing his address and got the same problem as he had had.

**Comment by OliverUK** [comment]
@NickWeir63  @lin-d-hop 
Although this isn't a critical issue, it makes a bad impression if shoppers experience this and no one is available to fix it in a reasonable time. I totally appreciate that we can't implement all fixes and improvements straight away but I do hope that when OFN UK launches properly, i.e. once hubs and shopfronts pay a fee, that there will be the support available to help with issues like the one above.

**Comment by OliverUK** [comment]
The same has now been flagged up by at least one other customer. My guess is it may be the ones imported as part of the data migration but I can't be certain.

**Comment by OliverUK** [comment]
Lynne, may I assign this to you for now?

**Comment by OliverUK** [comment]
@NickWeir63 I've removed the login details from your above comment to be on the save side in terms of security. Hope that's ok.
Strange that github allows me to edit another persons comment.

**Comment by NickWeir63** [comment]
@aidandunsdon sorry to bother you but if you have time to give us your opinion on why the imported shoppers might not be able to save updates to their addresses we would be very grateful.  Thanks

**Comment by lin-d-hop** [comment]
@OliverUK Sorry I haven't had a chance to look at this yet. The reason is because on first glance it looks like a tricky one to track down and may well spiral into something big. It is on my radar.

**Comment by lin-d-hop** [comment]
@OliverUK 
Think I tracked this down. You were right that it was related to the data import.
I've tested on the original shopper (email address was posted on the original bug report) and it worked fine. I updated their address to 123 Test St so you might like to log in as them and update to their actual address or to let them know and thank them for their help in tracking down the issue.

The fix should work for all users. Let me know if you experience any issues in the system of customer addresses not appearing where they should, it may be related to this fix.
